components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/LifeStatusController.java

```diff
@@ -58,9 +58,7 @@
 
     private static final String DECEASED = "deceased";
 
-    private static final String UNBORN = "unborn";
```

Comment:
...the reason we never mentioned it is probably because everyone forgot we have already added it